examples/computer_vision/byol_pytorch/README.md

```diff
@@ -0,0 +1,14 @@
+# Bootstrap Your Own Latent
```

Comment:
let's follow the README format in other examples to outline the file structure of this example, instructions on how to run, and expected results.

Reply:
Mostly done.  Still working on ImageNet results.